src/licensedcode/data/licenses/openssl-nokia-psk-contribution.yml

```diff
@@ -0,0 +1,6 @@
+key: openssl-nokia-psk-contribution
+name: OpenSSL Nokia PSK (Pre-Shared Key) Contribution
+short_name: OpenSSL Nokia PSK Contribution
+category: Permissive
+owner: Nokia
+spdx_license_key: LicenseRef-OpenSSL-Nokia-PSK-Contribution
```

Comment:
For now the approach has been not to put any LicenseRef for the license that are not known in the SPDX license list. There is a pending ticket #1328 by @sschuberth to review this approach and possibly assign always a LicenseRef (and there is also #532

Reply:
Thanks! Removed the `LicenseRef` and updated the PR.

It seems I looked at the only `.yml` underneath `licenses` containing such `LicenseRef`, e.g. `./src/licensedcode/data/licenses/here-proprietary.yml`. Out-of-scope here: should the LicenseRef be removed there?